.github/workflows/ci.yaml

```
@@ -0,0 +1,27 @@
+name: Continuous Integration
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ v1-mvp ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [18.x]
```

```suggestion
        node-version: [18.x, 20.x]
```
I've been on v20 for a while. Any reason both are not here? (I also messaged you in Discord)

---

18 is the latest stable, 20 is the latest nightly. See here: https://nodejs.org/en

We can use 20.x if you want.